style.css

```diff
@@ -0,0 +1,70 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins&display=swap");
+
+* {
+  font-family: "Poppins", sans-serif;
+}
+
+@media only screen and (max-width: 600px) {
+  /* navbar starts */
+  .navbar {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .logo {
+    padding-left: 6%;
+  }
+
+  .union-bar {
+    padding-right: 6%;
+    padding-top: 1.2%;
+  }
+
+  .logo a {
+    color: #6070ff;
+    text-decoration: none;
+    font-weight: 700;
+    font-size: 1.5rem;
+  }
+
+  /* navbar ends */
+
+  /* headline starts */
+  .headline {
```

Comment:
- [ ] It's great to see your project. However, I noticed that the content of the headline section is not aligned using flexbox. Please note that it is a requirement that the content of the headline section should be aligned using flexbox. Kindly add the necessary CSS properties to align the content properly. Let me know if you have any questions or concerns.